internal/server/data/migrations_test.go

```diff
@@ -158,6 +173,7 @@ func TestMigrations(t *testing.T) {
 		{
 			label: testCaseLine("202206281027"),
 			setup: func(t *testing.T, db WriteTxn) {
+				t.Skip("this migration no longer works with transactions")
```

Comment:
Should add a TODO here to fix later

Reply:
Ya, I wonder if we should fix this, or remove it since it's an old migration.